packages/jest-environment/src/index.ts

```diff
@@ -241,7 +241,10 @@ export interface Jest {
    */
   retryTimes(
     numRetries: number,
-    options?: {logErrorsBeforeRetry?: boolean},
+    options?: {
+      logErrorsBeforeRetry?: boolean;
+      retryFilter?: Circus.TestRetryFilter;
```

Comment:
Would be nice to have documentation here too. See above. Not sure which is better – one piece of prose, or a comment for each option?

Reply:
I suspect there might be more options in the future -- what do you think about just a comment for each option? It seems like potentially a bit more scalable